src/main/asciidoc/reference/r2dbc-repositories.adoc

```diff
@@ -118,6 +118,9 @@ interface ReactivePersonRepository extends ReactiveSortingRepository<Person, Lon
 The annotated query uses native bind markers, which are Postgres bind markers in this example.
 ====
 
+Note that the columns of a select used in a `@Query` annotation must match the names generated by the `NamingStrategy` for the respective property.
+If for a property no matching column is provided in a select, that property is not set. If that property is required by the persistence constructor, `null` is provided as a value or the default value for primitive types.
```

Comment:
Change to:

If a select statement does not include a matching column, that property is not set. If that property is required by the persistence constructor, either `null` or (for primitive types) the default value is provided.